examples/profile/keys/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Is this just excluding everything, but then overriding that with including the .gitignore? Is it simpler if we just exclude this directory in the parent .gitignore?

Reply:
It's a way to include the directory, but exclude everything within it, rather than use a "gitkeep"

Reply:
Ok, happy to use this instead of `.gitkeep`